src/components/inputs/rich-text-input/rich-text-input.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import requiredIf from 'react-required-if';
+import Types from 'slate-prop-types';
+import { Editor as SlateEditor } from 'slate-react';
+import { RenderMarkPlugin, RenderBlockPlugin } from './plugins';
+import UndoPlugin from './plugins/undo';
+import RedoPlugin from './plugins/redo';
+import ListPlugin from './plugins/list';
+import MarkPlugin from './plugins/mark';
+import Editor from './editor';
+
+const plugins = [
+  MarkPlugin({
```

Comment:
I'm not sure if this pattern is better than just dumping this all into one file 🤔

Reply:
Maybe we could have `defaultPlugins` on a file and use it here as the `defaultProp` of `plugins`, and allow users to pass their own list of `plugins`.

Reply:
Yeah I guess it depends if we want to make it configurable...? 🤔